lib/iterator.js

```diff
@@ -183,6 +183,33 @@ class Iterator {
     return new SkipWhileIterator(this, predicate, thisArg);
   }
 
+  // Consumes an iterator, partitioning it into Arrays.
```

Comment:
Same as for the previous PR, remove the dots
```suggestion
  // Consumes an iterator, partitioning it into Arrays
```